scvi/model/base/_base_model.py

```diff
@@ -224,6 +224,39 @@ def _register_manager_for_instance(self, adata_manager: AnnDataManager):
         instance_manager_store = self._per_instance_manager_store[self.id]
         instance_manager_store[adata_id] = adata_manager
 
+    def deregister_manager(self, adata_manager: Optional[AnnDataManager] = None):
```

Comment:
should this take the manager or the anndata, or both? I feel like most users aren't aware of how to get the manager object, and you could do with just the Anndata object by checking and fetching it's uuid

Reply:
hmm yeah since this is user-facing, I feel like this should just take the anndata